matRad_DijSampling.m

```diff
@@ -35,25 +34,63 @@
 %
 % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-if ~exist('SamplingRate','var')
-    SamplingRate       = 0.1;
+boolFineClustering  = false;                                     % flag to switch between fine and coarse radiological depth resolution
```

Comment:
I would remove the FineClustering Option. If you want to increase resolution you can also do that by decreasing deltaRadDepth. As you are rounding the radDepths to integers you have a minimum deltaRadDepth that makes sense of 1mm anyways...